.github/workflows/build.yml

```
@@ -16,83 +15,18 @@ on:
       - 'README.md'
       - 'CHANGELOG.md'
       - '.gitignore'
-      - '.env.example'
       - '.gitattributes'
       - 'infection.json.dist'
       - 'psalm.xml'
 
-  schedule:
-    - cron: '0 0 * * *'
-
 name: build
 
 jobs:
-  tests:
-    name: PHP ${{ matrix.php }}-${{ matrix.os }}
-
-    env:
+  build:
+    uses: yiisoft/actions/.github/workflows/codeception.yml@codeception
```

```suggestion
    uses: yiisoft/actions/.github/workflows/codeception.yml
```

Need to remember to change after merge PR in `yiisoft/actions`.